add fetch to popular movies and keywords

6ara6ac opened the pull request:
Импортировал axios, нужно установить зависимости в проекте. В функции кейвордс не знаю класс инпута, поэтому взял его типом текст, позже как будет html поменяю

6ara6ac committed:
add fetch to popular movies and keywords

Импортировал axios, нужно установить зависимости в проекте. В функции кейвордс не знаю класс инпута, поэтому взял его типом текст, позже как будет html поменяю

Myron5 reviewed (commented):
keyword приймаємо як параметр функції

Myron5 reviewed (commented):
keyword приймаємо як параметр функції + нам потрібно повернути totalPages крім results

Myron5 reviewed (commented):
якщо немає data або виникли помилки повертаємо null

6ara6ac committed:
added length check, param keyword of movies and extended request

6ara6ac committed:
add fetch by id

проверку не делал, так как если вводить id которого не существует выдает ошибку, по данным или статусу запроса не проверишь, так как функция не выполняется
